### DISCUSSION TITLE
Tooltip and button stays active with modal window

### DISCUSSION THREAD

**Discussion by mikrosmile** [filamentphp/filament#3638] [Help]
I noticed one issue:
I have a table where I edit records in modal window, when I cancel editing / or saving, the button "Edit" and tooltip stays active.
Is this normal behavior? 

https://user-images.githubusercontent.com/2136612/186328822-53305448-03b8-4be2-a60a-5fda8051094a.mov

**Comment by danharrin** [comment]
Yeah I think its normal because that button is still focused. Maybe Alpine refocuses it after the modal closes as an accessibility thing.
